src/core/ComboBox/ComboBox.tsx

```diff
@@ -436,6 +435,7 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
             autoCapitalize='none'
             autoCorrect='off'
             {...inputProps}
+            onChange={onInput}
```

Comment:
You said that this change would make sure the two onChanges are combined properly, but it seems to me like this would just make our onChange override theirs?

Reply:
If you expand whole file, you will see that our `onInput` handler are calling user's `onChange`:
![image](https://user-images.githubusercontent.com/36186912/159674554-57f1b3fc-2c8d-4276-8fb6-db24b7f0a824.png)